### DISCUSSION TITLE
Will there ever be an English version of the website.

### DISCUSSION THREAD

**Discussion by Alatarius** [aipictors/aipictors#1] [質問箱 - Q&A]
I really like the artwork on the site but the notice is that if you use a translator you basically can't sign up. This also extends to the discord servers as well. I am unable to learn more about the site nor can I engage with other artist nor be able to support / donate to them as I don't read or write the language.

**Comment by nakatsuka-k** [comment]
Thank you for liking our site.
We plan to provide translation support in the future.

**Comment by shigurenimo** [comment]
Add Engilish pages by adding the path `/en/` like this
- https://beta.aipictors.com/stickers
- https://beta.aipictors.com/en/stickers

issue: https://github.com/aipictors/aipictors/issues/3
